department-of-veterans-affairs/va.gov-cms
# VACMS-5887: Adding workaround for core issue related to block lazy loading

## Description
Closes #5887

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/126333482-8be85ee6-a5ec-48dc-9bd6-d187a1d82b5e.png)
![image](https://user-images.githubusercontent.com/2404547/126333496-68cd5577-2dfa-4857-af8e-235a79372943.png)

## QA steps
 - Go to /escanaba-vet-center and visually verify left sidebar is not visible.
 - Go to /pittsburgh-health-care and visually verify left sidebar is visible.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
